site/docs/red-team/index.md

```diff
@@ -237,3 +246,21 @@ That view includes a breakdown of specific test types that are connected to the
 ## Detailed guide
 
 See [the full guide](/docs/guides/llm-redteaming) for detailed info on configuring more complex prompts, dynamically generated prompts and RAG/chain/agents, and more.
+
+## redteam promptfooconfig.yaml
```

Comment:
this probably belongs in detailed guide or even a separate `red-team/configuration.md`